flask_annex/base.py

```diff
@@ -25,3 +25,6 @@ def save_file(self, key, in_file):
 
     def send_file(self, key):
         raise NotImplementedError()
+
+    def get_upload_info(self, key):
```

Comment:
Is this a good name? Should it instead be `get_save_info` to parallel `save_file`? `get_receive_info` to parallel `send_file`?